.gitattributes

```diff
@@ -0,0 +1,6 @@
+/test export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.travis.yml export-ignore
+phpunit.xml.dist export-ignore
+infection.json.dist
```

Comment:
I think there is an `export-ignore` missing here

Reply:
🦅👓